apiserver/plane/app/views/workspace/quick_link.py

```
@@ -0,0 +1,40 @@
+        if serializer.is_valid():
+            serializer.save(workspace_id=workspace.id, owner=request.user)
+            return Response(serializer.data, status=status.HTTP_201_CREATED)
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+    
+    
+    @allow_permission([ROLE.ADMIN, ROLE.MEMBER, ROLE.GUEST], level="WORKSPACE")
+    def partial_update(self, request, slug, pk):
+        quick_link = WorkspaceUserLink.objects.filter(pk=pk).first()
+
+        if quick_link: 
+            serializer = WorkspaceUserLinkSerializer(quick_link, data=request.data, partial=True)
```

Ruff (0.8.2)

Line too long (97 > 88)

(E501)